projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -243,6 +243,8 @@ public static function enqueue_scripts() {
 				'lifecycleStats'         => array(
 					'jetpackPlugins'            => self::get_installed_jetpack_plugins(),
 					'historicallyActiveModules' => \Jetpack_Options::get_option( 'historically_active_modules', array() ),
+					'ownedProducts'             => Products::get_products_by_ownership()['owned'],
+					'unownedProducts'           => Products::get_products_by_ownership()['unowned'],
```

Comment:
What do you think if pass the type of ownership as an argument so we can retrieve the specific values that we want at any time? Alternatively, if we are only going to call this method here, we can store the return value in a variable and use it afterwards:
```php
$products_by_ownership = Products::get_products_by_ownership();
//...

'ownedProducts'   => $products_by_ownership['owned'],
'unownedProducts' => $products_by_ownership['unowned'],
```

Reply:
Good idea, I'll get that changed, thank you!